tests/admin_views/test_multidb.py

```diff
@@ -68,6 +72,17 @@ def test_add_view(self, mock):
                 )
                 mock.atomic.assert_called_with(using=db)
 
+    @mock.patch("django.contrib.admin.options.transaction")
+    def test_get_add_view(self, mock):
+        for db in self.databases:
+            with self.subTest(db=db):
+                Router.target_db = db
+                self.client.force_login(self.superusers[db])
+                self.client.get(
+                    reverse("test_adminsite:admin_views_book_add"),
+                )
```

Comment:
```suggestion
    def test_read_only_methods_add_view(self, mock):
        for db in self.databases:
            with self.subTest(db=db):
                Router.target_db = db
                self.client.force_login(self.superusers[db])
                add_url = reverse("test_adminsite:admin_views_book_add")
                self.client.get(add_url)
                self.client.options(add_url)
                self.client.head(add_url)
                self.client.trace(add_url)
```
I think for each test we can add all of the methods